test/Microsoft.Buld.Sql.Tests/DotnetTestBase.cs

```diff
@@ -5,14 +5,15 @@
 using System.Diagnostics;
 using System.IO;
 using System.Text;
+using Microsoft.Build.Construction;
```

Comment:
Is this addition necessary?  Maybe I'm missing something you're using from this namespace.